src/java.base/share/classes/sun/nio/fs/Cancellable.java

```diff
@@ -117,7 +118,7 @@ public final void run() {
      * thread by writing into the memory location that it polls cooperatively.
      */
     static void runInterruptibly(Cancellable task) throws ExecutionException {
-        Thread t = new Thread(null, task, "NIO-Task", 0, false);
+        Thread t = InnocuousThread.newThread("NIO-Task", task);
```

Comment:
"NIO-Task" doesn't make sense as the thread name here, something like "CancellableOp" is closer to what it really is.

As you are there, the javadoc link to runInterruptibly is missing the "#", maybe we can fix this while we are here.

Reply:
I just left the name as it was but will change it as suggested and fix the link.